src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--day-1"></div>
```

Comment:
I believe that you could complete this task without modifiers for every element 🙂